openmetadata-ui/src/main/resources/ui/src/components/common/rich-text-editor/RichTextEditorPreviewer.tsx

```diff
@@ -32,13 +32,30 @@ const RichTextEditorPreviewer = ({
 }: PreviewerProp) => {
   const { t } = useTranslation();
   const [content, setContent] = useState<string>('');
-  const [hideReadMoreText, setHideReadMoreText] = useState<boolean>(
-    markdown.length <= maxLength
+
+  // initially read more will be false
+  const [readMore, setReadMore] = useState<boolean>(false);
+
+  // read more toggle handler
+  const handleReadMoreToggle = () => setReadMore((pre) => !pre);
+
+  // whether has read more content or not
+  const hasReadMore = useMemo(
+    () => enableSeeMoreVariant && markdown.length > maxLength,
```

Comment:
```suggestion
    () => enableSeeMoreVariant || markdown.length > maxLength,
```

Reply:
@chirag-madlani , we have to check for both conditions here, as we only want to show the read more/less button if `enableSeeMoreVariant` is true and the markdown length is greater than the max length.